src/components/AppText.tsx

```diff
@@ -27,7 +27,7 @@ export type AppTextProps = {
   bold?: boolean
   semiBold?: boolean
   medium?: boolean
-  color?: FontColor | GlobalColor
+  color?: FontColor | GlobalColor | string
```

Comment:
This was added so that I can have white texts on a button that has a fixed color (accent) no matter the theme.